pkg/resources/usercluster-webhook/deployment.go

```diff
@@ -182,3 +192,96 @@ func DeploymentCreator(data webhookData) reconciling.NamedDeploymentCreatorGette
 		}
 	}
 }
+
+func getEnvVars(data webhookData) ([]corev1.EnvVar, error) {
```

Comment:
this is the same as the machine controller one. I think we also need to add it to the usercluster-controller as the resource-quota calculation controller also uses the machine getMachineResourceUsage function

Reply:
can we create a separate pr for this?